addons/web/static/src/views/relational_model.js

```diff
@@ -532,6 +575,17 @@ export class Record extends DataPoint {
         return this._changes.map((change) => this.activeFields[change]);
     }
 
+    get formattedRecord() {
+        const record = Object.create(this, Object.getOwnPropertyDescriptors(this));
+        for (const fieldName in this.activeFields) {
+            record[fieldName] = {
+                value: getValue(this, fieldName),
+                raw_value: getRawValue(this, fieldName),
+            };
+        }
+        return record;
+    }
+
```

Comment:
@ged-odoo @aab-odoo What do you think of this addition? (following ged's comment about those attributes being present on the record).
It is needed to access some values from the templates like: record.activity_id.value, record.[...].raw_value, etc.
Before, it was only done in the KanbanRecord component, and should have been done also for the ActivityRecord.
It is now present in a consistent way to be used for both views, without adding it to the whole record instance